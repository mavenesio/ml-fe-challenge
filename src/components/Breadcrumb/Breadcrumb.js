import React from 'react';
import styled from 'styled-components';


const BreadcrumbContainer = styled.div`
  font-size:14px;
  margin: 16px 0px;
  font-family: Arial, Helvetica, sans-serif;
  background-color:#EEEEEE;
`;

const Breadcrumb = ({categories}) => (
    <BreadcrumbContainer>
        {categories && <>{categories.join(" > ")}</> }
    </BreadcrumbContainer>
    );

export default Breadcrumb;