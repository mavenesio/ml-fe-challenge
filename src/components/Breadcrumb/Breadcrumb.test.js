import React from 'react';
import Breadcrumb from './Breadcrumb';

describe('Breadcrumb', () => {
  it('should render without props', () => {   
    expect(<Breadcrumb />).toMatchSnapshot();
  });

  it('should render with props', () => {   
    expect(<Breadcrumb categories={['cat 1', 'cat 2']} />).toMatchSnapshot();
  });
});