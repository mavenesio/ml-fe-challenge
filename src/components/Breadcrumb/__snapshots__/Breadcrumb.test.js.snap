// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render the Breadcrumb Component correctly 1`] = `<styled.div />`;

exports[`Title should render with props 1`] = `<Breadcrumb />`;

exports[`Title should render without parameters 1`] = `<Breadcrumb />`;

exports[`Title should render without props 1`] = `<Breadcrumb />`;
